plugins/net-forward.yaml

```diff
@@ -0,0 +1,39 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: net-forward
+spec:
+  version: "v1.0.3"       
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+    uri: https://github.com/antitree/krew-net-forward/releases/download/v1.0.3/net-forward_v1.0.3.tar.gz
+    sha256: "645adb05d9ad68ecc91d9034cb80b1e2fe4762864323c12de0f2bc5afe20baa0"
+    bin: "net-forward"  
+  shortDescription: "Extend the functionality of port-forward connect to any IP and port (not just Pods and Services)."
+  homepage: https://github.com/antitree/krew-net-forward
+  caveats: |
+    This plugin needs the alpine:socat image available to the node.
```

Comment:
Is this a real caveat? How could this ever fail, unless it's removed from docker hub?

Reply:
I think this is still accurate because if you were to ever work on a cluster that only uses a private image registry and/or blocks access to Dockerhub for example (which in a security context might be a valid design decision), the ops may choose to only allow certain images that they build and not alpine:socat. In that case it would fail.